machine_configuration/100-intel-qat-intel-iommu-on.yaml

```diff
@@ -5,11 +5,13 @@ apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfig
```

Comment:
Can you change the commit log to:

machine_config: set intel_iommu=on for the worker nodes with QAT device

Signed-off-by: Hersh Pathak hersh.pathak@intel.com

Reply:
Updated, thanks for the review @uMartinXu